apix-impl/src/integration-test/java/eu/xenit/apix/tests/search/FtsNodeVisitorTest.java

```diff
@@ -54,7 +54,7 @@ public void TestTermALL() {
     public void TestTermEscaped() {
         String t = "type";
         SearchSyntaxNode node = builder.term(t, "my\"Val").create();
-        Assert.assertEquals(t.toUpperCase() + ":\"my\"Val\"", toFts(node));
```

Comment:
Please keep as many tests as possible out of integration test, which is **the** biggest bottleneck in APIX build time.
Please move this to the unit tests. The best solution would to move this entire class (I think that should be be possible without mocking).